### PR TITLE
Adds help to CLI

### LINKED ISSUE #21: CLI doesn't work if no argument is passed
So, what I think this will do, if I pass it two strings:
cat dog
Is that it will join them together to produce catdog

What you could do is check that args length > 0, then use args[0] and pass that through to the function.

If args length == 0, you could print out some help instructions and exit?

### PR DESCRIPTION
**Pull request by Cronan**
CLI changed to be `wsc`
Also exits if characters are not provided

Also fixes #21

### CODE REVIEW

File: src/WordscapesCheat.CLI/Program.cs [hunk 1]
```diff
@@ -8,12 +8,26 @@
...
+            // \t is a tab space
+            const string HelpMessage = "###Wordscapes Cheater###\n\n" + 
+                "Returns matching words based on the characters you provide.\n\n" + 
+                "wsc characters\n\n  characters  The characters that Wordscapes gives you.\n\n" + 
+                "Example:\nwsc cat\nact\ncat";
+
+            if (ArgsNeedsHelp(args))
+            {
+                Console.WriteLine(HelpMessage);
+                return;
```
Comment (Cronan) [inline src/WordscapesCheat.CLI/Program.cs]: This is where it exits

File: src/WordscapesCheat.CLI/Program.cs [hunk 2]
```diff
@@ -29,8 +43,16 @@ public static void PrintMatchingWords(string[] matchingWordsArray)
...
                 }
-            }            
+            }
         }
 
+        private static bool ArgsNeedsHelp(string[] args)
+        {
+            string[] HelpArgs = new[] { "/?", "--help", "/help", "?" };
+
+            return (args == null || args.Length == 0) //no args at all
```
Comment (Cronan) [inline src/WordscapesCheat.CLI/Program.cs]: The test whether to display help and exit includes a few special strings (e.g. `/?`) but also check sfor nulls and empty

File: src/WordscapesCheat.CLI/WordscapesCheat.CLI.csproj [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <ProjectGuid>{FF29FD59-7363-4B26-A24B-92FDDD8A2E51}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <RootNamespace>WordscapesCheat.CLI</RootNamespace>
-    <AssemblyName>WordscapesCheatCLI</AssemblyName>
+    <AssemblyName>wsc</AssemblyName>
```
Comment (Cronan) [inline src/WordscapesCheat.CLI/WordscapesCheat.CLI.csproj]: The CLI is now just called wsc

**Comment by KeenanCronyn** [comment]
This is really cool, thank you. 

To make sure I've understood this, you've included a new method called "ArgsNeedsHelp", which allows people to post something close to a help request (help, ?, etc.) and it displays a help message.

Just to make sure, you also changed the assembly name to wsc to make it easier to call via the command line right?

**Comment by Cronan** [comment]
Yeah, that’s exactly what I did

**Pull request merged**